plugin.video.ad6/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.ad6" name="Alpe d&#39;HuZes" version="0.1.0" provider-name="Menco Bolt">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.routing" version="0.2.1"/>
+
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>audio video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">

```

Comment:
We need summary and description in english for the addon website and addon browser. Please add new elements with `lang=en_GB`

Reply:
Done